src/api/helpers/map-CIS-countries/map-CIS-country/no-insurance-support/index.test.ts

```diff
@@ -0,0 +1,116 @@
+import noInsuranceSupport from '.';
+import { EXTERNAL_API_DEFINITIONS } from '../../../../constants';
+
+const {
+  CIS: {
+    COUNTRY_RATINGS,
+    ESRA_CLASSIFICATION: { VERY_HIGH, HIGH, STANDARD, NONE },
+    SHORT_TERM_COVER: { UNLISTED, CILC },
+  },
+} = EXTERNAL_API_DEFINITIONS;
+
+const mockInitCountry = {
+  esraClassification: NONE,
+  shortTermCover: UNLISTED,
+};
+
+const createMockParams = (countryRating: string, esraClassification?: string, shortTermCover?: string) => {
+  const params = {
+    ...mockInitCountry,
+    countryRating,
+  };
+
+  if (esraClassification) {
+    params.esraClassification = esraClassification;
+  }
+
+  if (shortTermCover) {
+    params.shortTermCover = shortTermCover;
+  }
+
+  return params;
+};
+
+const params = {
+  firstConditions: [
+    {
+      country: createMockParams(COUNTRY_RATINGS.A[0]),
+      description: 'A',
+      expectation: true,
+    },
+    {
+      country: createMockParams(COUNTRY_RATINGS.B[0]),
+      description: 'B',
+      expectation: true,
+    },
+    {
+      country: createMockParams(COUNTRY_RATINGS.C[0]),
+      description: 'C',
+      expectation: true,
+    },
+    {
+      country: createMockParams(COUNTRY_RATINGS.D[0]),
+      description: 'D',
+      expectation: true,
+    },
+    {
+      country: createMockParams(COUNTRY_RATINGS.NOT_APPLICABLE),
+      description: 'NOT_APPLICABLE',
+      expectation: true,
+    },
+    {
+      country: createMockParams('not recognised'),
+      description: 'not recognised',
+      expectation: false,
+    },
+  ],
+  secondConditions: [
+    {
+      country: createMockParams(COUNTRY_RATINGS.NOT_APPLICABLE, STANDARD, CILC),
+      description: 'STANDARD',
+      expectation: true,
+    },
+    {
+      country: createMockParams(COUNTRY_RATINGS.NOT_APPLICABLE, HIGH, CILC),
+      description: 'HIGH',
+      expectation: true,
+    },
+    {
+      country: createMockParams(COUNTRY_RATINGS.NOT_APPLICABLE, VERY_HIGH, CILC),
+      description: 'VERY_HIGH',
+      expectation: true,
+    },
+
+    {
+      country: createMockParams(COUNTRY_RATINGS.NOT_APPLICABLE, NONE, CILC),
+      description: 'NONE',
+      expectation: true,
+    },
+  ],
+};
+
+describe('helpers/map-CIS-countries/map-CIS-country/no-insurance-support', () => {
+  describe(`when shortTermCover is ${UNLISTED} and esraClassification is ${NONE}`, () => {
+    describe.each(params.firstConditions)('', ({ country, description, expectation }) => {
```

Comment:
should it be ''? after (params.firstConditions)

Reply:
I'm not following - `params.firstConditions` is defined in the test so no need for a condition on this?

Reply:
sorry I mean the empty string for the describe